electron-app/main/menu.ts

```diff
@@ -85,4 +86,99 @@ if (process.platform === 'darwin') {
   MENU.push(HELP_MENU);
 }
 
-export default MENU;
+export function popupContextMenu(
```

Comment:
I think it would be nice to break this out into its own file, and leave menu as-is.